data-tool/flows/tombstone/tombstone_utils.py

```diff
@@ -210,22 +221,160 @@ def format_resolutions_data(data: dict) -> list[dict]:
     return formatted_resolutions
 
 
-def get_snapshot_data_formatters() -> dict:
+def format_filings_data(data: dict) -> list[dict]:
+    # filing info in business
+    business_update_dict = {}
+
+    filings_data = data['filings']
+    formatted_filings = []
+    last_state_filing_idx = -1
+    idx = 0
+    for x in filings_data:
+        event_file_type = x['event_file_type']
+        # TODO: build a new complete filing event mapper
+        filing_type, filing_subtype = get_target_filing_type(event_file_type)
+        # skip the unsupported ones
+        if not filing_type:
+            continue
+
+        effective_date = x['f_effective_dt_str']
+        if not effective_date:
+            effective_date = x['e_event_dt_str']
+
+        meta_data = {
+            'colinFilingInfo': {
+                'eventType': x['e_event_type_cd'],
+                'filingType': x['f_filing_type_cd']
+            }
+        }
```

Comment:
when we add the placeholder filings, can we also add the following to the root of filngs.meta_data?

`isLedgerPlaceholder: true`